src/core/track.cpp

```diff
@@ -570,7 +569,7 @@ void trackContentObjectView::mousePressEvent( QMouseEvent * _me )
 							"a copy." ),
 					embed::getIconPixmap( "hint" ), 0 );
 		}
-		else if( m_autoResize == false )
+		else if( m_tco->getAutoResize() == false )
```

Comment:
Not sure what our project's preferences are here (@diizy can hopefully clarify), but I would personally tend to avoid a comparitor on a true/false (for readiblity purposes only).

```cpp
else if( !m_tco->getAutoResize() )
```